_assets/javascripts/components/bitmovin.js

```diff
@@ -11,8 +11,9 @@ class BitmovinManager {
     this.dateStringFormat = "YYYY/MM/DD HH:mm:ss";
     this.timeouts = [];
     this.container = document.getElementById(`${bitmovinConfig.id}`);
-    this.countdown = new CRDS.Countdown();
-
+    if (bitmovinConfig.countdown !== false)
+      this.countdown = new CRDS.Countdown();
+    
```

Comment:
Ooof. This syntax is nasty. How about something like this? /cc @j-brammer 

```js
if (bitmovinConfig.countdown !== false) {
  this.countdown = new CRDS.Countdown();
}
```

Reply:
I've become accustomed to seeing single line ifs. I will concede that it might be more obvious for someone later on to see it. Fix incoming